readme.md

```diff
@@ -4,6 +4,10 @@ Providing you with all sorts of Good Eggs-style fake data!
 
 All exported functions are stateless (meaning you don't have to worry about binding to `this`).
 
+## Purpose and Scope
+
```

Comment:
(Sorry for putting comments on random lines – I want to be able to have a proper thread for each comment...)

To me, a fundamental purpose of fake-eggs is to generated **realistic** or **plausible** fake data. This is as opposed to using a generic fake data library (directly), which would generate more general purpose categories of fake data. Could we document this as well?